vars/shared-OpenData.var.yml

```diff
@@ -49,13 +49,13 @@ extensions:
       url: "https://github.com/ckan/ckanext-dcat.git"
       version: "v1.2.0"
 
-    CKANExtScheming: &CKANExtScheming
-      name: "ckanext-scheming-{{ Environment }}"
-      shortname: "ckanext-scheming"
-      description: "CKAN Extension for Scheming"
-      type: "git"
-      url: "https://github.com/ckan/ckanext-scheming.git"
-      version: "release-3.0.0"
+    #~ CKANExtScheming: &CKANExtScheming
```

Comment:
why was this removed?

Reply:
It's already a transitive dependency of ckanext-validation, and for some reason adding it directly causes a clash on the test containers where it unsuccessfully tries to replace the distribution version of `six`. I've tried a bunch of ways to tell it to Not Do That, but none seemed to work. Letting it be installed transitively worked.